data/ocicrypt_keyprovider.conf

```diff
@@ -0,0 +1,7 @@
+{
+        "key-providers": {
+                "sample_kbs": {
```

Comment:
Doesn't the `sample_kbs` use `attestation-agent` as the keyprovider name?

Reply:
Yes, I use the old `sample_kbs` branch from jiale, I found latest `sample_kbs` Readme all use `attestation-agent` as the keyprovider name. Thanks, I'll fix that.